int8  mode
int16 VX
int16 angle
int16 VZ
int16 vx
int16 vy
int16 vz